sdk/netapp/arm-netapp/src/models/index.ts

```diff
@@ -679,16 +699,6 @@ export interface VolumePatch extends BaseResource {
   exportPolicy?: VolumePatchPropertiesExportPolicy;
 }
 
-/**
- * List of Mount Targets
- */
-export interface MountTargetList {
```

Comment:
why is this removed?

Reply:
I think this is expected.https://github.com/Azure/azure-rest-api-specs/commit/adf5808c3dcd1171579975f2874efc4f4f41f0ba#diff-4f53ea72e2650e22c8b0ee2770c55654L2432 they remove the definition in their previous swagger PR